canonical/ubuntu.com
# Cred exit survey

## Done

- [List of work items including drive-bys - remember to add the why and what of this work.]

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Go to [/credentials/exit-survey](https://ubuntu-com-12363.demos.haus/credentials/exit-survey)
- Fill out the form
- Filled-out data is stored in [this sheet](https://docs.google.com/spreadsheets/d/1L-e0pKXmBo8y_Gv9_jy9P59xO-w4FnZdcTqbGJPMNg0/edit?usp=sharing)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/CRED-248

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
